### ISSUE TITLE
Feature Suggestion - Drop out and use different activation function in each layer.

### ISSUE THREAD

**Issue by Goldflower** [tensorflow/playground#50]
I thought add these features are needed for all types of neuron networks, so it would be nice to add them.

In addition, maybe we could switch between feedforward and backpropagation mode?

**Comment by emmanuelrouxfr** [comment]
Dear @goldflower, by reading your issue it seems that you may appreciate the fork we've been working on !

https://coolearning.github.io/playground/

The Coolearning interface allows real-time network interaction during training, and much more ! Like individual learning rate, activation functions and regularization for each neuron... you can achieve your request by selecting the neurons of a layer and apply them a specific activation function !

See you there :)
